cli/main.rs

```diff
@@ -585,6 +587,21 @@ async fn cache_command(
   Ok(0)
 }
 
+async fn check_command(
+  flags: Flags,
+  check_flags: CheckFlags,
+) -> Result<i32, AnyError> {
```

Comment:
This function is superfluous. Rename cache_command to check_command.

Reply:
I'm not sure I agree, see https://github.com/denoland/deno/pull/14072#discussion_r835952423